src/move/econia/sources/user.move

```diff
@@ -3271,11 +3300,132 @@ module econia::user {
 
     // Test-only functions >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
 
+    #[test_only]
+    /// Immutably borrow market event handles for a market account.
+    inline fun borrow_market_event_handles_for_market_account_test(
+        market_id: u64,
+        user: address,
+        custodian_id: u64
+    ): &MarketEventHandlesForMarketAccount
+    acquires MarketEventHandles {
+        let market_event_handles_map_ref =
+            &borrow_global<MarketEventHandles>(user).map;
+        let market_account_id = get_market_account_id(market_id, custodian_id);
+        table::borrow(market_event_handles_map_ref, market_account_id)
+    }
+
+    #[test_only]
+    /// Return a `ChangeOrderSizeEvent` with the indicated fields.
+    public fun create_change_order_size_event_test(
```

Comment:
Why create these helpers? Having them makes it seem like there's something more interesting going on than... just building the struct. Don't see why these (create_change_order_size_event_test, create_place_limit_order_event_test, create_place_market_order_event_test) are here.

Reply:
These helpers are here so that the structs can be created from a call inside the `market` module

Move doesn't allow structs to be created from outside the module in which they are defined, so the `user` module needs to have public APIs that the market module can use

The events are emitted at the user level, but need to be tested in market-level scenarios (hence nearly all tests for this PR are in `market` module)